res/controllers/Hercules-DJControl-Inpulse-300-script.js

```diff
@@ -185,11 +376,358 @@ DJCi300.scratchWheel = function(channel, control, value, status, _group) {
     }
 };
 
-// Bending on the jog wheel (rotating using the edge)
-DJCi300.bendWheel = function(channel, control, value, _status, _group) {
-    var interval = DJCi300._convertWheelRotation(value);
-    engine.setValue(
-        "[Channel" + channel + "]", "jog", interval * DJCi300.bendScale);
+// Helper function that calculates samples per beat
+DJCi300._samplesPerBeat = function(deck) {
+    const sampleRate = engine.getValue(`[Channel${  deck  }]`, "track_samplerate");
+    const bpm = engine.getValue(`[Channel${  deck  }]`, "local_bpm");
+    // For some reason, multiplying by 60 makes the size 1/2 as large as it's supposed to be
+    // Hence, we multiply by 120 instead
```

Comment:
I don't quite understand. Are you arguing that returning 2x the audio sampling frequency is correct behavior of the CO or that its a bug?

Reply:
I originally put the comment there because I wasn't sure why minutes per beat had to be multiplied by 2*60 instead of just 60 to get the "correct" seconds per beat value. But if it's the sample rate instead of the framerate, then that makes sense. I'll change the comment to reflect that.